backend/schema/source/extra.prisma

```diff
@@ -135,6 +140,11 @@ type Subscription {
   goalLinkCreated(workspaceId: ID!): GoalLink!
   goalLinkDeleted(workspaceId: ID!): DeletedGoalLinkItem!
 
+  # ObjectiveLink subscriptions
+  objectiveLinkCreated(workspaceId: ID!): ObjectiveLink!
+  objectiveLinkUpdated(workspaceId: ID!): ObjectiveLink!
+  objectiveLinkDeleted(workspaceId: ID!): DeletedGoalLinkItem!
```

Comment:
That type should probably be renamed to something other than `DeletedGoalLinkItem`